grails-app/controllers/au/org/emii/aatams/detection/DetectionController.groovy

```diff
@@ -22,6 +22,11 @@ class DetectionController extends ReportController
         doList("detection")
     }
 
+    protected def doCount(resultList, queryName)
+    {
+        []
```

Comment:
The return value from this method isn't used. So this could just be an empty method.

ie.
<pre>protected def doCount(resultList, queryName)
{
}</pre>